package.json

```diff
@@ -141,5 +141,9 @@
   "engines": {
     "node": ">=14.5.0",
     "yarn": ">=1.21.0"
+  },
+  "volta": {
+    "node": "14.5.0",
+    "yarn": "1.21.0"
```

Comment:
Could we use version 1.19.0 instead?

version 1.21.0 has some bugs and yarn team already said they won't fix or maintain this version anymore because they are working on v2

https://github.com/yarnpkg/yarn/issues/7807

https://contentful.slack.com/archives/C029BE16E/p1628608153028200?thread_ts=1628607388.028100&cid=C029BE16E

Reply:
sure thing, done